test/blackbox-tests/test-cases/dune-jbuild-var-case/run.t

```diff
@@ -0,0 +1,18 @@
+All builtin variables are lower cased in Dune:
+
+  $ dune runtest --root dune-lower
+  Entering directory 'dune-lower'
+
+  $ dune runtest --root dune-upper
+  Entering directory 'dune-upper'
+  File "dune", line 3, characters 41-46:
+  Error: Uppercase variables are removed in dune files. Use: %{make}
+  [1]
+
+jbuilder retains the old names:
```

Comment:
Since it's about the file syntax rather than the tool, I think this should say "jbuild files" (and the commands can use `dune runtest`).